package_installs.R

```diff
@@ -47,4 +52,4 @@ library(torch)
 install_torch(reinstall = TRUE)
 
 # The R Keras package must be reinstalled after installing it in the python virtualenv.
-install.packages("keras")
\ No newline at end of file
+install_version("keras", version = "2.3.0.0", ask=FALSE)
```

Comment:
Are you sure the version is `2.3.0.0` and not `2.3.0`?

Reply:
Yes: 

```
> install_version("keras", version = "2.3.0", ask=FALSE)
Error in download_version_url(package, version, repos, type) : 
  version '2.3.0' is invalid for package 'keras'
> install_version("keras", version = "2.3.0.0", ask=FALSE)
Downloading package from url: http://cran.us.r-project.org/src/contrib/Archive/keras/keras_2.3.0.0.tar.gz
[...]
```